# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../root/asimage-reg-clobbered.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- root/asimage/Module.mk.orig	2004-10-29 18:07:32.000000000 +0200
+++ root/asimage/Module.mk	2004-12-01 16:05:26.755494376 +0100
@@ -66,6 +66,7 @@
 			gunzip -c $(ASTEPVERS).tar.gz | tar xf -; \
 		fi; \
 		cd $(ASTEPVERS); \
+		patch -p1 < ../asimage.patch; \
 		unset MAKEFLAGS; \
 		nmake -nologo -f libAfterImage.mak \
 		CFG="libAfterImage - Win32 Release")
@@ -83,6 +84,7 @@
 			fi; \
 		fi; \
 		cd $(ASTEPVERS); \
+		patch -p1 < ../asimage.patch; \
 		ACC=; \
 		ACFLAGS="-O"; \
 		if [ "$(CC)" = "icc" ]; then \
--- root/asimage/src/asimage.patch~	1970-01-01 01:00:00.000000000 +0100
+++ root/asimage/src/asimage.patch	2004-12-01 13:45:53.165473552 +0100
@@ -0,0 +1,10 @@
+--- libAfterImage/asimage.c~	2004-10-09 00:25:36.000000000 +0200
++++ libAfterImage/asimage.c	2004-12-01 13:44:55.275274200 +0100
+@@ -133,7 +133,6 @@
+                       "popl %%eax\n\t"
+                       : "=m" (mmx_available)
+                       : /* no input */
+-					  : "ebx", "ecx", "edx"
+ 			  );
+ #endif
+ 	return mmx_available;
